core/search/groups.go

```diff
@@ -8,12 +8,11 @@ import (
 
 	"github.com/nyaruka/mailroom/core/models"
 	"github.com/nyaruka/mailroom/runtime"
-	"github.com/olivere/elastic/v7"
 )
 
 // PopulateSmartGroup calculates which members should be part of a group and populates the contacts
 // for that group by performing the minimum number of inserts / deletes.
-func PopulateSmartGroup(ctx context.Context, rt *runtime.Runtime, es *elastic.Client, oa *models.OrgAssets, groupID models.GroupID, query string) (int, error) {
```

Comment:
just a little simplification - es isn't needed as separate arg because it's part of the runtime